src/libs/actions/IOU.js

```diff
@@ -686,17 +687,13 @@ function payIOUReport({
 }) {
     Onyx.merge(ONYXKEYS.IOU, {loading: true, error: false});
 
-    const payIOUPromise = paymentMethodType === CONST.IOU.PAYMENT_TYPE.EXPENSIFY
-        ? DeprecatedAPI.PayWithWallet({reportID, newIOUReportDetails})
-        : DeprecatedAPI.PayIOU({reportID, paymentMethodType, newIOUReportDetails});
-
     // Build the url for Paypal.me if they have selected it instead of a manual settlement or Expensify Wallet
     let url;
     if (paymentMethodType === CONST.IOU.PAYMENT_TYPE.PAYPAL_ME) {
         url = buildPayPalPaymentUrl(amount, requestorPayPalMeAddress, currency);
     }
```

Comment:
Do we need to handle the Paypal payment method here? I only see `payIOUReport` is used when the payment method is Expensify.

Reply:
@mollfpr this is deprecated code, it only handles paying using the Expensify wallet which is also going to be removed soon and we can remove this whole method.